cfme/configure/configuration/__init__.py

```diff
@@ -305,219 +304,129 @@ class ServerLogDepot(Pretty):
         ServerLogDepot.Credentials.clear()
 
     """
-    elements = UIRegion(
-        locators={
-            "last_message": InfoBlock("Basic Info", "Last Message"),
-            "last_log_collection": InfoBlock("Basic Info", "Last Log Collection"),
-        },
-    )
-
-    class Credentials(Updateable, Pretty):
-        """ This class represents the credentials for log depots.
 
-        Args:
-            p_type: One of ftp, nfs, or smb.
-            uri: Hostname/IP address of the machine.
-            username: User name used for logging in (ftp, smb only).
-            password: Password used for logging in (ftp, smb only).
-
-        Usage:
-
-            log_credentials = ServerLogDepot.Credentials("nfs", "backup.acme.com")
-            log_credentials.update()
-            log_credentials = ServerLogDepot.Credentials(
-                "smb",
-                "foobar",
-                "backup.acme.com",
-                username="jdoe",
-                password="xyz",
-            )
-            log_credentials.update()
-
-        """
-        pretty_attrs = ['p_type', 'name', 'uri', 'username', 'password']
-
-        server_collect_logs = Form(
-            fields=[
-                ("type", {
-                    version.LOWEST: Select("select#log_protocol"),
-                    '5.5': AngularSelect('log_protocol')}),
-                ("name", {
-                    version.LOWEST: None,
-                    "5.4": Input("depot_name")}),
-                ("uri", Input("uri")),
-                ("user", Input("log_userid")),
-                ("password", MultiFill(Input("log_password"), Input("log_verify"))),
-            ]
+    def __init__(self, p_type, depot_name=None, uri=None, username=None, password=None):
+        self.p_type = p_type
+        self.depot_name = depot_name
+        self.uri = uri
+        self.username = username
+        self.password = password
+
+        p_types = dict(
```

Comment:
**Optional**
Doesn't look like p_types needs to be part of the class, and could be a module var. Naming the parameter more descriptively than `p_type` would be nice, not very clear what 'p' is, even in context.